### ISSUE TITLE
Command `__vsc_escape_value` not found when using inshellisense under zsh

### ISSUE THREAD

**Issue by B4night** [microsoft/inshellisense#136]
**Describe the bug**
1. When I use _is_ to activate _inshellisense_, the shell will start a new page and the fist line is `zsh: __vsc_escape_value: command not found...`
2. After activating the environment, no matter what command I use, the error `zsh: __vsc_escape_value: command not found...` will always follow the output of the command

``` shell
➜  ~ ls
Desktop  Documents  Downloads  Music  Pictures  Project  Public  snap  Software  Templates  Videos  Zotero
zsh: __vsc_escape_value: command not found...                                                                                             
➜  ~ cd ..
zsh: __vsc_escape_value: command not found...                                                                                           
➜  /home cd - 
~
zsh: __vsc_escape_value: command not found...                                                                                                                                            
➜  ~ ls
Desktop  Documents  Downloads  Music  Pictures  Project  Public  snap  Software  Templates  Videos  Zotero
zsh: __vsc_escape_value: command not found... 
```

**To Reproduce**
1. use command `is`
2. type any command

**Expected behavior**
Provide a patch to fix this or tutorial to enable this command

**Environment**
 - OS: Fedora39  6.5.12-300.fc39.x86_64
 - Output of `is --version`: 0.0.1-rc.6
 - Nodejs Version: v20.10.0
 - Shell: zsh
 - Shell Version: zsh 5.9 (x86_64-redhat-linux-gnu)

**Comment by alanhg** [comment]
app is Terminal or WebStorm?

**Comment by B4night** [comment]
> app is Terminal or WebStorm?

Terminal, I am using konsole and the version is 23.08.3

**Comment by jacquesg** [comment]
Also happens with kitty, seeing a similar message

```
__is_update_cwd:1: command not found: __vsc_escape_value
```

**Comment by cpendery** [comment]
Once that fix in #137 is merged, you can rebuild from source via the following command.
```
git clone https://github.com/microsoft/inshellisense.git && cd inshellisense && npm install && npm run build && npm link && cd ..
```
